pkg/chunk/grpc/grpc_client.go

```diff
@@ -0,0 +1,25 @@
+package grpc
+
+import (
+	"flag"
+
+	"google.golang.org/grpc"
+)
+
+// Config for a StorageClient
+type Config struct {
+	Address string `yaml:"address,omitempty"`
```

Comment:
We should keep naming consistency between YAML and CLI flags. Either we rename this to `server_address` or the CLI flag to `grpc-store.address`.